frontend/src/metabase/dashboard/components/DashboardInfoSidebar/DashboardInfoSidebar.unit.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import userEvent from "@testing-library/user-event";
+import { renderWithProviders, screen } from "__support__/ui";
+import { createMockDashboard } from "metabase-types/api/mocks";
+import { setupRevisionsEndpoints } from "__support__/server-mocks/revision";
+import type { Dashboard } from "metabase-types/api";
+
+import { DashboardInfoSidebar } from "./DashboardInfoSidebar";
+
+interface SetupOpts {
+  dashboard?: Dashboard;
+}
+
+function setup({ dashboard = createMockDashboard() }: SetupOpts = {}) {
+  const setDashboardAttribute = jest.fn();
+  // TODO: Needs to change this name after https://github.com/metabase/metabase/pull/32357 is merged
+  const saveDashboardAndCards = jest.fn();
+
+  setupRevisionsEndpoints([]);
+
+  renderWithProviders(
+    <DashboardInfoSidebar
+      dashboard={dashboard}
+      setDashboardAttribute={setDashboardAttribute}
+      saveDashboardAndCards={saveDashboardAndCards}
+    />,
+  );
+
+  return {
+    setDashboardAttribute,
+  };
+}
+
+describe("DashboardInfoSidebar", () => {
+  it("should render the component", () => {
+    setup();
+
+    expect(screen.getByText("About")).toBeInTheDocument();
+  });
+
+  it("should allow to set description", () => {
+    const { setDashboardAttribute } = setup();
+
+    userEvent.click(screen.getByTestId("editable-text"));
+    userEvent.type(
+      screen.getByPlaceholderText("Add description"),
+      "some description",
+    );
+    userEvent.tab();
+
+    expect(setDashboardAttribute).toHaveBeenCalledWith(
+      "description",
+      "some description",
+    );
+  });
+
+  it("should allow to clear description", () => {
```

Comment:
I have verified that this test would fail on `master`. Which is good since it captures the bug correctly.